### DISCUSSION TITLE
interactive.subgraph建立子图报错

### DISCUSSION THREAD

**Discussion by zjarles** [alibaba/GraphScope#3188] [Q&A]
sub_graph=interactive.subgraph("g.V().has('trmnl','trmnl_black_flag',1).bothE('register')") 
在建立子图报错：GremlinServerError: 500: Status{code=UNAVAILABLE, description=Network closed for unknown reason, cause=null}
Loading vertex labeled _ and edge labeled _:   0%|          | 0/10 [00:00<?, ?it/s]2023-09-06 14:46:05,939 [INFO][coordinator:546]: Clean up resources, cleanup_instance: True, is_dangling: False

**Comment by BingqingLyu** [comment]
Hi, could you please provide more info, including:
1. the statistics of the dataset (e.g., number of vertices, edges, the number of subgraph edges etc. in case if the subgraph is too large that occurs OOM); 
2. the way you startup graphscope (including configuration);

**Reply by zjarles** [reply]
不是点数量的问题，是不能实现3至4跳的子图，只能实现outE一跳的子图。但是一跳子图，通过表直接sql关联，就可以得到。只有多跳子图才更有意义。能否实现这句话呢 interactive.subgraph（“g.V（）.has（'trmnl_black_flag'，1）.both（'3..4'，'register'）”）

**Comment by zjarles** [comment]
您好，
利用这个语句sub_graph=interactive.subgraph("g.V().has('trmnl_black_flag',1).outE('register')")，可以建立子图成功。
利用3..4跳这种语句，就不能了
sub_graph=interactive.subgraph("g.V().has('trmnl_black_flag',1).both('3..4','register')")
报错如下：
GremlinServerError: 500: Status{code=UNKNOWN, description=execution_error: user error: kind(Unknown), caused&nbsp;
by:Unexpected data type in exec neither vertex nor edge in GraphWriter, cause=null}


请问，这是什么原因呢？

**Reply by BingqingLyu** [reply]
Hello,

The reason you're encountering this issue is that the query results are in the form of paths, and currently, our system does not support constructing a subgraph based on the path data type. If this feature is crucial for your use case, please let us know, and we will consider incorporating support for it into our development roadmap.
